routes/web.php

```diff
@@ -14,3 +14,6 @@
 */
 
 Route::get('/', [\App\Http\Controllers\HomeController::class, 'index'])->name('home');
+
+Route::get('/celebs', [\App\Http\Controllers\CelebController::class, 'index'])->name('celebs');
```

Comment:
After doing some research, I notice that the best practice is to name this as 'celebs.index'